homeassistant/components/goalzero/binary_sensor.py

```diff
@@ -21,23 +21,23 @@
 BINARY_SENSOR_TYPES: tuple[BinarySensorEntityDescription, ...] = (
     BinarySensorEntityDescription(
         key="backlight",
-        name="Backlight",
+        translation_key="backlight",
         icon="mdi:clock-digital",
     ),
     BinarySensorEntityDescription(
         key="app_online",
-        name="App online",
+        translation_key="app_online",
         device_class=BinarySensorDeviceClass.CONNECTIVITY,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
     BinarySensorEntityDescription(
         key="isCharging",
-        name="Charging",
+        translation_key="is_charging",
```

Comment:
This translation key can be removed, the device class is giving the translation here.